fast/stages/02-networking-nva/variables.tf

```diff
@@ -69,13 +69,13 @@ variable "l7ilb_subnets" {
     region        = string
   })))
   default = {
-    prod = [
-      { ip_cidr_range = "10.128.92.0/24", region = "europe-west1" },
-      { ip_cidr_range = "10.128.93.0/24", region = "europe-west4" }
-    ]
     dev = [
-      { ip_cidr_range = "10.128.60.0/24", region = "europe-west1" },
-      { ip_cidr_range = "10.128.61.0/24", region = "europe-west4" }
+      { ip_cidr_range = "10.128.159.0/24", region = "europe-west1" },
```

Comment:
Per my comment, use the third to last /24 available on the carved IP estate.
Also applies to all following ranges.